lgsm/functions/check_deps.sh

```diff
@@ -261,6 +264,9 @@ fn_found_missing_deps(){
 			elif [ -n "$(command -v yum 2>/dev/null)" ]; then
 				cmd="sudo yum -y install ${array_deps_missing[@]}"
 				eval "${cmd}"
+			elif [ -n "$(command -v pkg 2>/dev/null)" ]; then
+				cmd="sudo pkg -y install ${array_deps_missing[@]}"
```

Comment:
I'm very confused as to why Codacy doesn't like this line... it is exactly the same as 3 lines above. Should I change this along with the existing lines to Codacy's suggestion?